### PR TITLE
Play dictionary

### LINKED ISSUE #991: [#2PlaysAMonth]: Dictionary App
### What's the Play about?

It's a Dictionary that you can play around with to understand meaning of new English Words.

### What ReactJS concept will be used to create this play?

ASYNC/AWAIT, JSX, State management, APIs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Abhishek-90**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #991 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

When word is invalid.

Screenshot_2

When word is valid and meaning is found.

Screenshot_1

# Demo
[Video](https://www.stack-stream.com/v/dictionary-1)

### CODE REVIEW

**Comment by Abhishek-90** [comment]
@atapas @koustov please review the changes!

File: src/plays/play-dictionary/PlayDictionary.css [hunk 1]
```diff
@@ -0,0 +1,181 @@
+/* enter stlyes here */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.main-container {
```
Comment (koustov) [inline src/plays/play-dictionary/PlayDictionary.css:8]: Please use our play specific class name for all the CSS selectors.
like `play-dict-main-container` , ` play-dict-container-input` etc

**Review by koustov — CHANGES_REQUESTED** [review]

**Review by koustov — CHANGES_REQUESTED** [review]
Please reduce the cover image size

**Comment by koustov** [comment]
Also you need to record a demo on stackstream and update the video url

File: src/plays/play-dictionary/Readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Play Dictionary
+
+Its a Dictionary that you can play around with to understand meaning of new English Words.
+
+## Play Demographic
+
+- Language: js
```
Comment (Sachin-chaurasiya) [inline src/plays/play-dictionary/Readme.md:7]:
```suggestion
- Language: JavaScript
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
Left a few comments, thanks

**Review by Sachin-chaurasiya — DISMISSED** [review]

**Comment by Abhishek-90** [comment]
> Also you need to record a demo on stackstream and update the video url

video url is added inside `Readme.md` file of the play folder

**Comment by Abhishek-90** [comment]
> Please reduce the cover image size

Cover size is reduced, if it is still not up to the mark, please give me a reference dimensions to adjust it accordingly.

File: src/plays/play-dictionary/PlayDictionary.css (hunk 1 above)
Comment (Abhishek-90) [reply src/plays/play-dictionary/PlayDictionary.css]: changes for class names and css selectors made as required.

**Comment by koustov** [comment]
> > Also you need to record a demo on stackstream and update the video url
> 
> video url is added inside `Readme.md` file of the play folder

Actually, we should update the play meta, which means, updating the URL in the video section when you are on the Edit Play page. Anyway, I have updated it for you. So everything looks good to me. Once I validate the build, will merge it.

**Review by koustov — APPROVED** [review]
lgtm

**Pull request merged**